packages/bridge-controller/jest.config.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * For a detailed explanation regarding each configuration property and type check, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+const merge = require('deepmerge');
+const path = require('path');
+
+const baseConfig = require('../../jest.config.packages');
+
+const displayName = path.basename(__dirname);
+
+module.exports = merge(baseConfig, {
+  // The display name when running multiple projects
+  displayName,
+
+  // An object that configures minimum threshold enforcement for coverage results
+  coverageThreshold: {
+    global: {
+      branches: 89,
```

Comment:
⏭️ We should strive to hit 100% test coverage. Otherwise, if we are on a branch and we get errors from Jest saying that coverage has not been met, it is painful to figure out which lines have been added in that branch that we need to test.